api/openapi/generate.go

```diff
@@ -0,0 +1,6 @@
+package openapi
+
+//go:generate go tool oapi-codegen -config config.yaml owner-server.yaml
```

Comment:
I would recommend that you try to generate the owner server code under its own owner package. In other words:
```
oapi-codegen -config config.yaml -package oapi_owner owner-server.yaml
```

We will need this eventually anyways in order to support three different ServiceInterface interfaces, as the REST API for each server will be different. 

With that each server is allowed to define a custom Server type tailored to its needs, which it will then pass to the generated Handler() function imported from its server-specific package.

The common types should continue to use the "openapi" package name since... well, they're common.

Reply:
I refactored to generate the owner server code in its own package, should be separated now